app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for human in people:
+        if "wife" in human and human["wife"] is not None:
+            Person(human["name"], human["age"]).wife = human["wife"]
```

Comment:
You could shorten check by using the walrus operator and `.get()` to perform the check:
```suggestion
        if wife_name := human.get("wife"):
            Person(human["name"], human["age"]).wife = wife_name
```